config/alphagov.yml

```diff
@@ -146,11 +146,11 @@ govuk-accounts-tech:
     - andysellick
     - barrucadu
     - bevanloon
-    - brucebolt
     - erino
     - huwd
     - danacotoran
     - jonathanglassman
+    - AlanGabbianelli
```

Comment:
It might be a good idea to keep this list in alphabetical order so it is easier to manage?